lab_2_retrieval_w_bm25/start.py

```diff
@@ -26,7 +29,7 @@ def main() -> None:
             documents.append(file.read())
     with open("assets/stopwords.txt", "r", encoding="utf-8") as file:
         stopwords = file.read().split("\n")
-    result = None
+    result = tokenize(documents[0])
```

Comment:
show here the full presentation of your work

Reply:
done